### PR TITLE
Add debugging email for tricky UCR bug

### PR DESCRIPTION
**Pull request by NoahCarnahan**
Trying to get a handle on this bug: http://manage.dimagi.com/default.asp?252832

@calellowitz @emord 
buddy @gcapalbo

### CODE REVIEW

File: corehq/apps/userreports/pillow.py [hunk 1]
```diff
@@ -182,6 +186,33 @@ def _rebuild_sql_tables(self, adapters):
                     except TableRebuildError as e:
                         _notify_rebuild(six.text_type(e), sql_adapter.config.to_json())
                 else:
+                    try:
+                        if sql_adapter.config.get_id in ("static-enikshay-episode", "static-np-migration-3-episode"):
```
Comment (dimagimon) [inline corehq/apps/userreports/pillow.py]: E501 line too long (117 > 115 characters)

File: corehq/apps/userreports/pillow.py [hunk 2]
```diff
@@ -1,9 +1,13 @@
 from __future__ import absolute_import
+
+import pprint
 from collections import defaultdict
 from datetime import datetime, timedelta
 import hashlib
 
 from alembic.autogenerate.api import compare_metadata
+from django.conf import settings
+from django.core.mail import send_mail
```
Comment (calellowitz) [inline corehq/apps/userreports/pillow.py]: is there a reason you used this instead of the `send_mail_async` from `corehq.apps.hqwebapp.tasks`

Comment (NoahCarnahan) [inline corehq/apps/userreports/pillow.py]: no reason. Updated.

**Review by emord — APPROVED** [review]

**Comment by NoahCarnahan** [comment]
no test coverage, so merging now

**Comment by NoahCarnahan** [comment]
ah, I was a little over eager on this, running relevant tests locally before deploying it

**Pull request merged**